controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -625,7 +625,15 @@ func (r *StorageClassClaimReconciler) reconcileCephBlockPool() error {
 			return fmt.Errorf("Could not find  device set named default in Storage cluster")
 		}
 		pgUnitSize := util.GetPGBaseUnitSize(deviceSet.Count)
-		r.cephBlockPool.Labels[controllers.StorageConsumerNameLabel] = r.storageConsumer.Name
+		if r.cephBlockPool.Labels == nil {
+			r.cephBlockPool.Labels = map[string]string{
+				controllers.StorageConsumerNameLabel: r.storageConsumer.Name,
+			}
+
+		} else {
```

Comment:
Remove this one duplicate line https://github.com/red-hat-storage/ocs-operator/blob/c1d20824e55fb233e2dc077d4ebbda60a07582e0/controllers/storageclassclaim/storageclassclaim_controller.go#L461-L463